frontend/package.json

```diff
@@ -18,6 +18,7 @@
     "@hookform/resolvers": "2.9.6",
     "@reduxjs/toolkit": "1.8.3",
     "clsx": "1.2.1",
+    "dayjs": "^1.11.4",
```

Comment:
Do not use ^ operator for specifying dependency version

Reply:
done